### DISCUSSION TITLE
Installing locally phoenix in debian 11

### DISCUSSION THREAD

**Discussion by itg00** [phcode-dev/phoenix#1043] [General]
Goo days 
Im trying to install phcode in linux to use it locally 
why?  i live in an area whit bad internet connection and need to work 

the instructions says 
* Install gulp globally once.  `npm install -g gulp-cli` (use **sudo** in *nix systems)
* run `npm install`
* To build after npm install: `npm run build`
 

but in in the second step 
'root@red:/home/usuario/Descargas/phoenix-main# npm install'  i have errors 

fatal: no es un repositorio git ( o ningún padre en el punto de montado /)
Parando en el límite del sistema de archivos (GIT_DISCOVERY_ACROSS_FILESYSTEM no establecido).

it says this is not a github repositorie ?
soo im doing wrong or im missing something ??

thanks for your time

**Comment by itg00** [comment]
i find a different set of instructions here 
https://github.com/phcode-dev/phoenix-desktop

but same problem at building 
npm run build

`(node:2782) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
[16:23:16] The following tasks did not complete: build, <series>, _patchAcornLib
[16:23:16] Did you forget to signal async completion?
`

**Comment by abose** [comment]
Phoenix supports limited offline mode, so just going to phcode.dev in the browser should work if phcode is loaded once in the browser.

Native builds are a work in progress, so the desktop repos are not ready yet.

It will be ready by June for full offline support and native builds.
